2025-10-02 20:46:18,856 INFO    MainThread:14216 [wandb_setup.py:_flush():81] Current SDK version is 0.22.1
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_setup.py:_flush():81] Configure stats pid to 14216
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_setup.py:_flush():81] Loading settings from C:\Users\Ery\.config\wandb\settings
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_setup.py:_flush():81] Loading settings from e:\PD1ModelTrainings\PD1ModelTrainingCodes\wandb\settings
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_init.py:setup_run_log_directory():705] Logging user logs to e:\PD1ModelTrainings\PD1ModelTrainingCodes\wandb\run-20251002_204618-kds9u0rk\logs\debug.log
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_init.py:setup_run_log_directory():706] Logging internal logs to e:\PD1ModelTrainings\PD1ModelTrainingCodes\wandb\run-20251002_204618-kds9u0rk\logs\debug-internal.log
2025-10-02 20:46:18,857 INFO    MainThread:14216 [wandb_init.py:monkeypatch_ipython():624] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001BEFD37C050>
2025-10-02 20:46:18,858 INFO    MainThread:14216 [wandb_init.py:init():832] calling init triggers
2025-10-02 20:46:18,858 INFO    MainThread:14216 [wandb_init.py:init():837] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-10-02 20:46:18,858 INFO    MainThread:14216 [wandb_init.py:init():880] starting backend
2025-10-02 20:46:21,188 INFO    MainThread:14216 [wandb_init.py:init():883] sending inform_init request
2025-10-02 20:46:21,264 INFO    MainThread:14216 [wandb_init.py:init():891] backend started and connected
2025-10-02 20:46:21,268 INFO    MainThread:14216 [wandb_run.py:_label_probe_notebook():1330] probe notebook
2025-10-02 20:46:21,271 INFO    MainThread:14216 [wandb_run.py:_label_probe_notebook():1340] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-10-02 20:46:21,271 INFO    MainThread:14216 [wandb_init.py:init():961] updated telemetry
2025-10-02 20:46:21,327 INFO    MainThread:14216 [wandb_init.py:init():985] communicating run to backend with 90.0 second timeout
2025-10-02 20:46:22,271 INFO    MainThread:14216 [wandb_init.py:init():1036] starting run threads in backend
2025-10-02 20:46:22,479 INFO    MainThread:14216 [wandb_run.py:_console_start():2506] atexit reg
2025-10-02 20:46:22,479 INFO    MainThread:14216 [wandb_run.py:_redirect():2354] redirect: wrap_raw
2025-10-02 20:46:22,479 INFO    MainThread:14216 [wandb_run.py:_redirect():2423] Wrapping output streams.
2025-10-02 20:46:22,479 INFO    MainThread:14216 [wandb_run.py:_redirect():2446] Redirects installed.
2025-10-02 20:46:22,483 INFO    MainThread:14216 [wandb_init.py:init():1076] run started, returning control to user process
2025-10-02 20:46:22,911 INFO    MainThread:14216 [wandb_init.py:_post_run_cell_hook():594] [no run ID] resuming backend
2025-10-02 20:48:48,398 INFO    MainThread:14216 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-10-02 20:48:48,398 INFO    MainThread:14216 [wandb_init.py:_pre_run_cell_hook():583] [no run ID] pausing backend
2025-10-02 20:48:48,403 INFO    MainThread:14216 [wandb_init.py:_post_run_cell_hook():594] [no run ID] resuming backend
2025-10-02 20:48:49,563 INFO    MainThread:14216 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-10-02 20:48:49,563 INFO    MainThread:14216 [wandb_init.py:_pre_run_cell_hook():583] [no run ID] pausing backend
2025-10-02 20:48:49,569 INFO    MainThread:14216 [wandb_init.py:_post_run_cell_hook():594] [no run ID] resuming backend
2025-10-02 20:48:50,160 INFO    MainThread:14216 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-10-02 20:48:50,160 INFO    MainThread:14216 [wandb_init.py:_pre_run_cell_hook():583] [no run ID] pausing backend
2025-10-02 20:48:50,164 INFO    MainThread:14216 [wandb_init.py:_post_run_cell_hook():594] [no run ID] resuming backend
2025-10-02 20:49:08,095 INFO    MainThread:14216 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-10-02 20:49:08,095 INFO    MainThread:14216 [wandb_init.py:_pre_run_cell_hook():583] [no run ID] pausing backend
2025-10-02 20:49:08,100 INFO    MainThread:14216 [wandb_init.py:_post_run_cell_hook():594] [no run ID] resuming backend
2025-10-02 20:49:12,858 INFO    MainThread:14216 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-10-02 20:49:12,859 INFO    MainThread:14216 [wandb_init.py:_pre_run_cell_hook():583] [no run ID] pausing backend
2025-10-02 20:49:12,878 INFO    MainThread:14216 [wandb_run.py:_finish():2272] finishing run eryyyj-technological-institute-of-the-philippines/pd1training/kds9u0rk
2025-10-02 20:49:12,879 INFO    MainThread:14216 [jupyter.py:save_history():480] not saving jupyter history
2025-10-02 20:49:12,879 INFO    MainThread:14216 [jupyter.py:save_ipynb():404] not saving jupyter notebook
2025-10-02 20:49:12,879 INFO    MainThread:14216 [wandb_init.py:_jupyter_teardown():609] cleaning up jupyter logic
2025-10-02 20:49:12,879 INFO    MainThread:14216 [wandb_run.py:_atexit_cleanup():2471] got exitcode: 0
2025-10-02 20:49:12,880 INFO    MainThread:14216 [wandb_run.py:_restore():2453] restore
2025-10-02 20:49:12,880 INFO    MainThread:14216 [wandb_run.py:_restore():2459] restore done
2025-10-02 20:49:14,012 INFO    MainThread:14216 [wandb_run.py:_footer_sync_info():3869] logging synced files
